static/src/javascripts/components/enhancer/enhancer.js

```diff
@@ -1,8 +1,4 @@
-define([
-    'require'
-], function (
-    require
-) {
+define(function () {
```

Comment:
It sounds like it might have been detected as a static require by mistake causing an inadvertent circular reference.

Try writing `require('m' + 'odule', cb)` and see if that works? If so, I'll update the regular expression this side with a fix for this case.